dianna/methods/rise_timeseries.py

```diff
@@ -54,10 +55,13 @@ def explain(self,
             model_or_function, preprocess_function=self.preprocess_function)
         self.masks = generate_masks(input_timeseries,
                                     number_of_masks=self.n_masks,
+                                    feature_res=self.feature_res,
                                     p_keep=self.p_keep)
-        masked = mask_data(input_timeseries, self.masks, mask_type=mask_type)
+        self.masked = mask_data(input_timeseries,
```

Comment:
Is it necessary to keep the masked data in the class? A copy of masked data could take some space since the input for timeseries is always the whole series (rather than one single instance in other cases).

Reply:
Good point! Storing it should be optional if at all. It can be convenient for a user to inspect the masks that were used, that's why it's in their in the first place.

Reply:
implemented optional storing of these variables